jina/executors/indexers/vector/ngt.py

```diff
@@ -53,9 +53,7 @@ def query(self, keys: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple['np.nd
         idx = []
         for key in keys:
             results = index.search(key, size=top_k, epsilon=self.epsilon)
-            index_k = []
-            distance_k = []
-            [(index_k.append(result[0]), distance_k.append(result[1])) for result in results]
+            index_k, distance_k = zip(*results)
```

Comment:
It's better to verify `results` is empty
```suggestion
            if results:
                index_k, distance_k = zip(*results)
                idx.append(index_k)
                dist.append(distance_k)
```

## Reference:
Index.search(): https://github.com/yahoojapan/NGT/blob/master/python/ngt/base.py#L363

Reply:
wait this is wrong